pkgs/by-name/la/lavacli/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitLab,
+  python3,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "lavacli";
+  version = "2.2.0";
+  pyproject = true;
+
+  src = fetchFromGitLab {
+    owner = "lava";
+    repo = "lavacli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Pfsf2a5y3rpoTGY3PmNdq9ojSjBVBfawz/LTQmIQ2vU=";
+  };
+
+  build-system = with python3.pkgs; [
+    setuptools
+  ];
+
+  dependencies = with python3.pkgs; [
+    aiohttp
+    jinja2
+    requests
+    ruamel-yaml
+    voluptuous
+  ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove with lib; as it increases eval times